apps/src/pythonlab/pyodideWebWorker.ts

```diff
@@ -37,15 +37,30 @@ async function loadPyodideAndPackages() {
   // matplotlib, which pythonlab_setup depends on, and numpy,
   // which will frequently be used. We have seen issues with loading these via
   // loadPyodide, so we load them here to ensure they are available.
-  await pyodide.loadPackage([
-    'numpy',
-    'matplotlib',
-    // These are custom packages that we have built. They are defined in this repo:
-    // https://github.com/code-dot-org/pythonlab-packages
-    `/blockly/js/pyodide/${version}/unittest_runner-0.1.0-py3-none-any.whl`,
-    `/blockly/js/pyodide/${version}/pythonlab_setup-0.2.0-py3-none-any.whl`,
-    `/blockly/js/pyodide/${version}/neighborhood-0.2.0-py3-none-any.whl`,
-  ]);
+  const loadErrors: string[] = [];
+  await pyodide.loadPackage(
+    [
+      'numpy',
+      'matplotlib',
+      // These are custom packages that we have built. They are defined in this repo:
+      // https://github.com/code-dot-org/pythonlab-packages
```

Comment:
Update where packages are defined.

Reply:
Any plans to deprecate `code-dot-org/pythonlab-packages`?

Reply:
Good call, will update. I'll file a ticket to deprecate the old package too!